vllm/core/block_manager.py

```diff
@@ -98,6 +98,41 @@ def get_num_free_blocks(self) -> int:
         return (self.num_blocks - self.current_num_blocks +
                 self.evictor.num_blocks)
 
+    def get_num_cached_blocks(self, seq: Sequence) -> int:
```

Comment:
Can you add doctsring to indicate the difference between cached vs computed blocks?

Reply:
Hi, thank you for your review! Since now it is very late in my area, I plan to add unit tests and more doctstrings tommorow. I have reviewed my logic, and think it shoud be right but kind of tricky. I will add more comments to make it more easy to understand. Thank you!